EasyPost.Tests/TestUtils.cs

```diff
@@ -24,7 +24,8 @@ public class TestUtils
             "keys",
             "phone_number",
             "phone",
-            "test_credentials"
+            "test_credentials",
+            "public_key"
```

Comment:
Please leave the `public_key` in the cassettes. The fact that it's public means it's not sensitive and intended for consumption. This could eventually backfire on us if we start scrubbing it.

Reply:
Resolved